tests/test_preprocessing.py

```diff
@@ -191,6 +192,46 @@ def test_replace_x_y_nominal_lat_lon(dask, nans):
     assert set(replaced_ds.lat.dims) == set(["x", "y"])
 
 
+def test_interp_nominal_lon():
+    """
+    Check that https://github.com/jbusecke/xMIP/issues/295 was fixed in https://github.com/jbusecke/xMIP/pull/296
+
+    In https://github.com/jbusecke/xMIP/blob/0270f4b4977d512adc2337d4a547b39e25d2f2da/tests/test_preprocessing.py,
+    the old issue was replicated (and illustrated that the tests would have failed then).
+    """
+
+    def _get_dummy_longitude() -> np.ndarray:
+        # Totally arbitrary data (although len(lon) has to be > 360 to see the issue)
+        lon = np.linspace(0, 360, 513)[:-1]
+
+        # Add some NaN values just as an example
+        lon[2 + 30 : len(lon) // 2 + 50] = np.nan
+        return lon
+
+    def _lons_parsed_make_sense(
+        input_lons: np.ndarray, lons_parsed: np.ndarray
+    ) -> bool:
+        """
+        Check if the parsed longitudes make sense.
+        Since we know that the input-lons are all monotonically increasing, the parsed lons should also do that.
+        """
+        accepted_differences_between_lon_coords = np.unique(np.diff(input_lons))
+        if len(accepted_differences_between_lon_coords) not in [1, 2]:
+            raise RuntimeError(
+                f"Cannot work with changed format of inputdata {accepted_differences_between_lon_coords}"
+            )
+        diff_pars_lons = np.unique(np.diff(lons_parsed))
+        return np.all(
+            [x in accepted_differences_between_lon_coords for x in diff_pars_lons]
+        )
+
+    lons = _get_dummy_longitude()
+    lons_parsed = _interp_nominal_lon(lons)
+    assert _lons_parsed_make_sense(
+        lons, lons_parsed
+    ), "Parsed lons after the fix of #296 are still bad?"
```

Comment:
The tests are now passing, so this comment confuses me slightly. Is it a leftover?

Reply:
That's the message that the `AssertionError` will produce once failed, see for example the message in https://github.com/jbusecke/xMIP/actions/runs/9857864431/job/27217936984 (pytest also prints the `lons`, and `lons_parsed` so it looks a bit messy).

Of course, it shouldn't (and doesn't) fail so I could as well remove it

Reply:
That would be awesome! Ill merge it right away.

Reply:
Great! I already pushed the commit